Some tweaks for checkpoint stress test

lutovich opened the pull request:
- warmup runs for 2 checkpoint intervals
- let worker threads run for 20 seconds before we start collecting throughput results
- print throughput results with timestamps
- require 60% of throughput results to be in one stddev from average
- increase default page cache memory from 2g to 4g

lutovich committed:
Some tweaks for checkpoint stress test

 * warmup runs for 2 checkpoint intervals
 * let worker threads run for 20 seconds before we start collecting throughput results
 * print throughput results with timestamps
 * require 60% of throughput results to be in one stddev from average
 * increase default page cache memory from 2g to 4g

lutovich commented:
@davidegrohmann would be nice if you could take a look at this PR.

davidegrohmann commented:
@lutovich looks fine

chrisvest commented on stresstests/src/test/java/org/neo4j/kernel/stresstests/transaction/checkpoint/CheckPointingLogRotationStressTesting.java:
```diff
@@ -81,15 +81,15 @@ public void shouldBehaveCorrectlyUnderStress() throws Throwable
                 .setConfig( GraphDatabaseSettings.mapped_memory_page_size, pageSize )
-                .setConfig( GraphDatabaseSettings.check_point_interval_time, CHECK_POINT_INTERVAL_SECONDS + "s" )
+                .setConfig( GraphDatabaseSettings.check_point_interval_time, CHECK_POINT_INTERVAL_MINUTES + "m" )
                 .setConfig( GraphDatabaseFacadeFactory.Configuration.tracer, "timer" )
```
I think you should also set `GraphDatabaseSettings.keep_logical_logs` to `false`, to prevent build-up of transaction log data for this build.

lutovich replied:
Good point. Added this setting now.

lutovich committed:
Do not keep tx logs in checkpoint stress test

They were the only thing that made store size larger over time. This could
make build agents to run out of disk space.
